scripts/xpp_update_happi_line

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+"""
+Updates items in the happi database to reflect the which line XPP is 
+using (mono line, pink line).
+
+Assumes pcds-conda, and grabs the currently active happi config.
```

Comment:
This is probably good enough for this little script. Just to note, some other scripts sometimes have a Python source code file along with a bash entrypoint that make the running environment consistent:
```bash
source pcds_env
cd $SCRIPT_PATH
python the_script.py "$@"
```

Reply:
good thought, I can add this.